eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -208,7 +208,7 @@
   <suppress checks="com.azure.tools.checkstyle.checks.ExternalDependencyExposedCheck"
             files="com.azure.core.http.okhttp.OkHttpAsyncHttpClientBuilder"/>
   <suppress checks="com.azure.tools.checkstyle.checks.ExternalDependencyExposedCheck"
-            files="com.azure.opentelemetry.exporters.azuremonitor.AzureMonitorExporter"/>
+            files="AzureMonitorExporter"/>
```

Comment:
is this correct with no package name here?

Reply:
Good catch! I think IDE refactoring removed the package name. Added the package name back.